src/docs/sections/instagram.hbs

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  {{> html-head title="Instagram" }}
+</head>
+<body>
+  <section class="instagram pt-4 text-center bg-primary-light">
+    <div class="col-lg-8 col-md-9 col-sm-10 col-12 text-center mx-auto pb-4">
```

Comment:
col classes must be always used along with container and row. if you need a full-width layout use container-fluid